<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintCenterManagerView</class>
 <widget class="QMainWindow" name="PrintCenterManagerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zilberman Publisher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons.qrc">
    <normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</normaloff>:/Icons/MainIcon/Resources/MainIcons/free-icon-book-4341050.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="name_lable">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="id_lable">
            <property name="text">
             <string>id: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="address_lable">
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="order_count_lable">
        <property name="text">
         <string>Выполнено работ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ваши партии (в работе)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="on_work_table"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="start_box">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_work">
            <property name="text">
             <string>Выполнить работы</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Ваши партии (напечатанные)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="ready_table"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="deny_box">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deny_work">
            <property name="text">
             <string>Отменить статус</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
